src/transformers/models/auto/modeling_flax_auto.py

```diff
@@ -141,13 +141,6 @@
     ]
 )
 
-FLAX_MODEL_FOR_SEQ_TO_SEQ_CAUSAL_LM_MAPPING = OrderedDict(
```

Comment:
Why did we remove this? We have this in PT as well no? The `run_summarization_flax` script uses this

Reply:
We had it twice, now `FLAX_MODEL_FOR_SEQ_TO_SEQ_CAUSAL_LM_MAPPING` only exist once :-)